android/linting/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
+}
+
+dependencies {
+    implementation("com.pinterest.ktlint:ktlint-cli-ruleset-core:0.49.1")
+    implementation("com.pinterest.ktlint:ktlint-rule-engine-core:0.49.1")
```

Comment:
Is it possible to migrate the following dependency versions to the [libs.versions.toml](https://github.com/opensrp/fhircore/blob/main/android/gradle/libs.versions.toml)  file?

Reply:
Do we need to enforce this version on the spotless ktlint config in the root [build.gradle.kts](https://github.com/opensrp/fhircore/blob/6dfa937ee22cb10c64850c716206efe7d96e4a22/android/build.gradle.kts#L43) file?